### PR TITLE
Compatibility with WP 5.9

### PR DESCRIPTION
**Pull request by mahdiyazdani**
This mini PR ensures compatibility with the latest WordPress release.

Linked task: https://app.asana.com/0/1200358610796599/1201721411659383/f

### CODE REVIEW

File: src/Edit.js [hunk 1]
```diff
@@ -64,6 +64,13 @@ import Controls from './components/Controls';
  */
 import Inspector from './components/Inspector';
 
+import useGetMediaType from './hooks/useGetMediaType';
```
Comment (gooklani) [inline src/Edit.js:67]: Would you please import the hook from `wp-react-hooks` 🙂?

**Review by gooklani — APPROVED** [review]
Thank you @mahdiyazdani 🙌. I have imported `useGetMediaType` from the `wp-react-hooks` on my local, and everything works fine.

File: src/Edit.js (hunk 1 above)
Comment (mahdiyazdani) [reply src/Edit.js]: Oh. I have just removed the line 🙂 Thank you 👍

**Review by kuserich — APPROVED** [review]
Very nice! Thank you, @mahdiyazdani

**Pull request merged**